app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+
+  has_many :categories, foreign_key: 'author_id', class_name: 'Categorie'
+  has_many :transaktions, foreign_key: 'author_id', class_name: 'Transaktion'
```

Comment:
- [ ] Please make sure that in your project there are validations for all user input. Because of the ubiquity of this data throughout your app, what might seem like a negligible error in user input could easily cascade into widespread data corruption problems or problems with app performance. I would strongly recommend you do it for all your user data. :+1: